docs/api/apiv3/paths/work_schedule_non_working_days.yml

```diff
@@ -0,0 +1,106 @@
+---
+get:
+  summary: Lists non-working days
+  operationId: list_non_working_days
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Lists non-working days for a given date interval.
+  parameters:
+    - name: include_weekend
+      in: query
+      description: |-
+        Include weekend days in the response when `true`.
+      required: false
+      schema:
+        default: false
+        type: boolean
+    - name: filters
+      in: query
+      description: |-
+        JSON specifying filter conditions.
+
+        Accepts the same format as returned by the [queries](https://www.openproject.org/docs/api/endpoints/queries/)
+        endpoint. Currently supported filters are:
+
+        + interval: The inclusive date interval to scope non working days to look up.
+
+          Example: `{ "interval": { "operator": "<>d", "values": ["2022-05-02","2022-05-26"] } }` would return non working days
+          between May 5 and May 26 2022, inclusive.
+      example: '[{ "interval": { "operator": "<>d", "values": ["2022-05-02","2022-05-26"] } }]'
+      required: false
+      schema:
+        type: string
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/non_working_day_collection_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+          example:
+            _type: Error
+            errorIdentifier: urn:openproject-org:api:v3:errors:InvalidRequestBody
+            message: The request body was not a single JSON object.
+post:
+  summary: Create a non working day
+  operationId: create_non_working_day
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Marks a day as being a non-working day.
+
+    Does nothing if the day is already a non-working day.
+  parameters:
```

Comment:
is there a specific reason why to put the request data into mandatory query params, instead of using the request body? I'd say, the body should be the first choice when transporting creation request data.

Reply:
That's because I learnt only very recently about `requestBody` :)
Thanks for pointing it out. I'll change it.

Reply:
Done